packages/cubejs-client-react/src/hooks/cube-api.js

```diff
@@ -0,0 +1,11 @@
+import { useContext } from 'react';
+import CubeContext from '../CubeContext';
+
+export function useCubeApi(explicitApiOrName) {
```

Comment:
This should only accept `name` as a param and default to the `default` api

Reply:
This is just a helper function to avoid code duplication, not new API. But I can change it if you like; it will just mean a bit more complexity on the caller side.